sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -878,6 +1182,20 @@ Exceptions:
 * Throws synchronously with error code `invalid` if `recordingQueue` is
   recording to a different graph.
 
+|
+[source,c++]
+----
+std::vector<node> get_nodes() const;
+----
+|Returns a list of all the nodes present in the graph.
```

Comment:
We should specify:
* If the list returned here has a consistent order, e.g. order nodes were added to the graph
* If the list order reflects some legal traversal order for execution